### PR TITLE
Install Dropbox on Fedora systems

### PR DESCRIPTION
**Pull request by neingeist**
Install Dropbox on Fedora systems using the official YUM/DNF package
repository:

https://linux.dropbox.com/fedora/

### CODE REVIEW

**Comment by neingeist** [comment]
Hey Julien,

any thoughts on this? :)

File: tasks/Fedora.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+- name: Install dropbox package repo
+  copy: src=dropbox.repo
+        dest=/etc/yum.repos.d/dropbox.repo
+  when: ansible_distribution_version|int <= dropbox_fedora_max_releasever
```
Comment (jdauphant) [inline tasks/Fedora.yml:5]: `tags: ["packages","dropbox"]` missing

File: tasks/Fedora.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+- name: Install dropbox package repo (fallback)
+  copy: src=dropbox.repo.f{{dropbox_fedora_max_releasever}}
+        dest=/etc/yum.repos.d/dropbox.repo
+  when: ansible_distribution_version|int > dropbox_fedora_max_releasever
```
Comment (jdauphant) [inline tasks/Fedora.yml:10]: `tags: ["packages","dropbox"]` missing

File: tasks/Ubuntu.yml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-- name: Download dropbox
+- name: Download dropbox .deb
```
Comment (jdauphant) [inline tasks/Ubuntu.yml]: Download dropbox Ubuntu package

**Comment by jdauphant** [comment]
Hello @neingeist ,
Thanks and sorry I have miss the email about the pull request :(
I have made some comments.
Can you had fedora support in the meta/main.yml ?
https://github.com/jdauphant/ansible-role-dropbox/blob/master/meta/main.yml

Best regards,
Julien

File: tasks/Fedora.yml (hunk 1 above)
Comment (neingeist) [inline tasks/Fedora.yml:5]: Do the tags in tasks/main.yml not suffice?

File: tasks/Ubuntu.yml (hunk 3 above)
Comment (neingeist) [inline tasks/Ubuntu.yml]: Fixed that one!

**Comment by neingeist** [comment]
So, that should do it, except for the tags. I think the tags in `tasks/main.yml` are sufficient but may be mistaken.

**Comment by jdauphant** [comment]
You have right for the tags, my mistake, we can cleanup useless tags.

File: tasks/Fedora.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  when: ansible_distribution_version|int <= dropbox_fedora_max_releasever
+
+- name: Install Fedora dropbox package repo (fallback)
+  copy: src=dropbox.repo.f{{dropbox_fedora_max_releasever}}
+        dest=/etc/yum.repos.d/dropbox.repo
+  when: ansible_distribution_version|int > dropbox_fedora_max_releasever
+
+- name: Install Fedora dropbox package
+  dnf: name=nautilus-dropbox state=present
+  tags: ["packages","dropbox"]
```
Comment (jdauphant) [inline tasks/Fedora.yml]: Unnecessary tags

File: tasks/Ubuntu.yml [hunk 5]
```diff
@@ -1,8 +1,8 @@
 ---
-- name: Download dropbox
+- name: Download Ubuntu dropbox package
   get_url: url={{ dropbox_url_ubuntu_deb }} dest={{dropbox_tmp_deb}}
   tags: ["packages","dropbox"]
```
Comment (jdauphant) [inline tasks/Ubuntu.yml]: Unnecessary tags

File: tasks/Ubuntu.yml [hunk 6]
```diff
@@ -1,8 +1,8 @@
...
 
-- name: Install dropbox for Debian OS family
+- name: Install Ubuntu dropbox package
   apt: deb={{dropbox_tmp_deb}}
   tags: ["packages","dropbox"]
```
Comment (jdauphant) [inline tasks/Ubuntu.yml]: Unnecessary tags

**Comment by neingeist** [comment]
Alright, the redundant tasks are now removed!

**Comment by jdauphant** [comment]
@neingeist Perfect, thanks :)

**Comment by neingeist** [comment]
Ah, I should mention that I left updating `galaxy_info` to you, regarding Ubuntu.

**Pull request merged**